### ISSUE TITLE
The battery_alert package has trouble with 0.170 HA release due to Groups changes

### ISSUE THREAD

**Issue by luckyjay** [notoriousbdg/Home-AssistantConfig#25]
Check out breaking changes for 0.107 https://www.home-assistant.io/blog/2020/03/18/release-107/ and the note related to Groups. Because the battery_alert package uses group.set, it's having some issues with this new release because that service no longer exists - at least, not with that name.

Not sure how to fix, but wanted to report the issue.

Error returned by HA is "Unable to find service group/set."

**Comment by luckyjay** [comment]
I just corrected an issue in my groups configuration and now it seems the service call is working okay, but I think there are still other issues because on the battery page there are a number of unavailable battery group entities.

**Comment by iMiMx** [comment]
https://www.home-assistant.io/integrations/group/

Lines such as 'view' and 'control' are no longer valid:

 battery_alert:
    control: hidden

  battery_view:
    view: yes

**Comment by iMiMx** [comment]
Seems like there is a pull request already that fixes this:

https://github.com/notoriousbdg/Home-AssistantConfig/pull/23/files

**Comment by IsaTTeN** [comment]
I have the same problem

**Comment by lpt2007** [comment]
Same problem here. Can someone update script to work 0.107+?

It is good script. It should be nice to see it in HASS COMMUNITY STORE ;)

**Comment by idfxken** [comment]
yeah, i made a merge request but think notorious is busy with other life stuff atm. you can just use an editor to find and delete the offending lines.

**Comment by schumi2004** [comment]
Not only does the group/view give issues but it also returns the following error for me when I toggle settings: Unable to find service input_number/turn_on
Home Assistant 0.110.3

**Comment by KptnKMan** [comment]
Hi everyone,
I got this to work with my HassOS install.

I made a PR here and added a little documentation:
https://github.com/notoriousbdg/Home-AssistantConfig/pull/27

Feel free to test, the direct raw file is here:
https://raw.githubusercontent.com/notoriousbdg/Home-AssistantConfig/164d3d736abfed943c4a4a71ec769022dc3045aa/packages/battery_alert.yaml

I'm interested to know if this works for others.

Thanks.
